js/tests/visual/tab.html

```diff
@@ -163,6 +163,45 @@
           <p>Trust fund seitan letterpress, keytar raw denim keffiyeh etsy art party before they sold out master cleanse gluten-free squid scenester freegan cosby sweater. Fanny pack portland seitan DIY, art party locavore wolf cliche high life echo park Austin. Cred vinyl keffiyeh DIY salvia PBR, banh mi before they sold out farm-to-table VHS viral locavore cosby sweater. Lomo wolf viral, mustache readymade thundercats keffiyeh craft beer marfa ethical. Wolf salvia freegan, sartorial keffiyeh echo park vegan.</p>
         </div>
       </div>
+
+      <h4>Tabs with nav (with fade)</h4>
+      <nav class="nav nav-pills">
+        <div class="nav-item">
+          <a class="nav-link active" data-toggle="tab" href="#home5">Home</a>
+        </div>
```

Comment:
Is it possible for `.nav-item` to be on the anchor? Example: `<a class="nav-item nav-link active">...</a>`?